tools/maplint/lints/multiz_cables.yml

```diff
@@ -0,0 +1,5 @@
+=/obj/structure/cable/multilayer/multiz:
+  banned_neighbors:
+  - /obj/structure/cable
+  - /obj/structure/cable/layer1
+  - /obj/structure/cable/layer3
```

Comment:
```suggestion
```
- /obj/structure/cable will already iterate through its subtypes so these lines won't do anything as they have already been checked for. If you want to get the strict paths you use the = operator like in =/obj/structure/cable/multilayer/multiz